src/js/components/AppLastTaskFailureComponent.jsx

```diff
@@ -0,0 +1,98 @@
+var React = require("react/addons");
+
+var AppsStore = require("../stores/AppsStore");
+var AppsActions = require("../actions/AppsActions");
+var AppsEvents = require("../events/AppsEvents");
+var TaskMesosUrlComponent = require("../components/TaskMesosUrlComponent");
+
+var AppLastTaskFailureComponent = React.createClass({
+  displayName: "AppLastTaskFailureComponent",
+
+  propTypes: {
+    appId: React.PropTypes.string.isRequired
+  },
+
+  getInitialState: function () {
+    return {
+      app: AppsStore.getCurrentApp(this.props.appId)
+    };
+  },
+
+  componentDidMount: function () {
```

Comment:
We can save this request, which can be very large, if there are a lot of tasks, because this is already called by the polling itself, every 5 seconds.
So you could remove componentDidMount() here completely.